examples/ptb/train_ptb.py

```diff
@@ -4,9 +4,13 @@
 This code is ported from the following implementation written in Torch.
 https://github.com/tomsercu/lstm
 
+Lines in this file are directly referenced by the RST file
+`docs/source/examples/ptb.rst`. The RST file must be updated in conjunction.
```

Comment:
When it should be updated is not stated.
It's better to align with the statement in `glance.py`.

Reply:
Thanks for the tip. Fixed as suggested.